lib/resources/majors.tsx

```diff
@@ -2425,144 +2479,238 @@ const bsCS_New: Major = {
+            'EN.500.112 Gateway Computing: JAVA <br /> ' +
+            'For equivalent ways to satisfy this requirement, contact your advisor and create a custom course which satisfies this requirement.',
+          required_credits: 3,
+          criteria: 'EN.500.112[C]',
+        },
+        {
+          description:
+            '<b>Intermediate Programming</b> <br /> ' +
+            'EN.601.220 Intermediate Programming',
+          required_credits: 4,
+          criteria: 'EN.601.220[C]',
+        },
+        {
+          description:
+            '<b>Data Structures</b> <br /> EN.601.226 Data Structures',
+          required_credits: 4,
+          criteria: 'EN.601.226[C]',
+        },
+        {
+          description:
+            '<b>Computer System Fundamentals</b> <br /> ' +
+            'EN.601.229 Computer System Fundamentals',
+          required_credits: 3,
+          criteria: 'EN.601.229[C]',
+        },
+        {
+          description:
+            '<b>Mathematical Foundations of Computer Science</b> <br /> ' +
+            'EN.601.230 Mathematical Foundations of Computer Science',
+          required_credits: 4,
+          criteria: 'EN.601.230[C]',
+        },
+        {
+          description:
+            '<b>Intro Algorithms</b> <br /> EN.601.433 Intro Algorithms',
+          required_credits: 3,
+          criteria: 'EN.601.433[C]',
           double_count: ['N/A'],
         },
         {
           description:
-            '<b>Upper Level Undergraduate: </b><p>12 upper level CS credits in addition to the required Algorithms course</p>',
+            '<b>Upper Level CS Credits</b> <br /> ' +
+            'At least 12 more upper level CS credits are required. ',
           required_credits: 12,
-          criteria:
-            'EN Computer Science[D]^AND^Upper Level Undergraduate[L]^NOT^EN.601.433[C]^NOT^EN.601.633[C]',
+          criteria: 'EN Computer Science[D]^AND^Upper Level[L]',
           double_count: [
-            '<b>One Team(CSCI-TEAM) designated course.</b><p> This Team course may overlap other course ' +
-              'requirements, for example to count as both Team and Software.</p>',
-            '<b>2 Upper Level Classifications:</b><p>At least one upper level course in two of these four different classification</p> ' +
-              'areas: Applications(CSCI-APPL), Systems(CSCI-SYST), Software(CSCI-SOFT) and Reasoning(CSCI-RSNG)',
+            '<b>Team Requirement</b> <br /> ' +
+              'Select one course with Program of Study Tag CSCI-TEAM.',
           ],
         },
         {
           description:
-            '<b>2 Upper Level Classifications:</b><p>At least one upper level course in two of these four different classification</p> ' +
-            'areas: Applications(CSCI-APPL), Systems(CSCI-SYST), Software(CSCI-SOFT) and Reasoning(CSCI-RSNG)',
+            '<b>CS Electives</b> <br /> ' +
+            'Six additional credits of Computer Science are required.' +
```

Comment:
super minor but insert space at the end haha

Reply:
Done